### ISSUE TITLE
Error on getMediaList after upgrading SDK to 1.0.9 

### ISSUE THREAD

**Issue by frederikheremans** [Bynder/bynder-java-sdk#8]
After upgrading to latest `1.0.9` build, we see errors when calling `getMediaList(..)`. Our code:

```
Response<List<Media>> mediaResponse = client.getAssetBankService().getMediaList(new MediaQuery().setKeyword(filterText)).blockingSingle();
```

The error that occurs (whenever the `filterText` passed results in at least one matching item):

```
java.lang.IllegalStateException: null
	at com.google.gson.JsonArray.getAsString(JsonArray.java:226) ~[gson-2.8.2.jar:na]
	at com.bynder.sdk.util.MediaTypeDeserializer.readMetapropertiesMap(MediaTypeDeserializer.java:60) ~[bynder-java-sdk-1.0.9.jar:na]
	at com.bynder.sdk.util.MediaTypeDeserializer.deserialize(MediaTypeDeserializer.java:41) ~[bynder-java-sdk-1.0.9.jar:na]
	at com.bynder.sdk.util.MediaTypeDeserializer.deserialize(MediaTypeDeserializer.java:27) ~[bynder-java-sdk-1.0.9.jar:na]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69) ~[gson-2.8.2.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.8.2.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.8.2.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.8.2.jar:na]
	at retrofit2.converter.gson.GsonResponseBodyConverter.convert(GsonResponseBodyConverter.java:37) ~[converter-gson-2.3.0.jar:na]
	at retrofit2.converter.gson.GsonResponseBodyConverter.convert(GsonResponseBodyConverter.java:25) ~[converter-gson-2.3.0.jar:na]
	at retrofit2.ServiceMethod.toResponse(ServiceMethod.java:119) ~[retrofit-2.3.0.jar:na]
	at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:218) ~[retrofit-2.3.0.jar:na]
	at retrofit2.OkHttpCall.execute(OkHttpCall.java:180) ~[retrofit-2.3.0.jar:na]
	at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:41) ~[adapter-rxjava2-2.3.0.jar:na]
	at io.reactivex.Observable.subscribe(Observable.java:12005) ~[rxjava-2.1.12.jar:na]
	at io.reactivex.internal.operators.observable.ObservableSingleMaybe.subscribeActual(ObservableSingleMaybe.java:30) ~[rxjava-2.1.12.jar:na]
	at io.reactivex.Maybe.subscribe(Maybe.java:4065) ~[rxjava-2.1.12.jar:na]
	at io.reactivex.Maybe.blockingGet(Maybe.java:2190) ~[rxjava-2.1.12.jar:na]
	at io.reactivex.Observable.blockingSingle(Observable.java:5239) ~[rxjava-2.1.12.jar:na]
	at com.wolfoil.compete.integration.service.impl.BynderServiceImpl.searchRootGroup(BynderServiceImpl.java:108) ~[classes/:na]
```

Without any modifications to our code (other than changing the version of the bynder-sdk version) this works on both `1.0.8` and `1.0.7`.

**Comment by frederikheremans** [comment]
By the way, using the latest 1.1.0 also causes the same exception.

**Comment by danielsequeira** [comment]
I found out about the issue today and already have a fix for it. I will release a new version between today and tomorrow

**Comment by danielsequeira** [comment]
we just released a new version 1.1.1, that should fix this issue
